main.js

```diff
@@ -1,47 +1,89 @@
-    newDiv.style.height=circleRadius+"px"
-    newDiv.classList.add("circle")
-    newDiv.style.left=(leftBounceDiv+randomIntFromInterval(0,diffRightLeft-circleRadius))+"px"
-    newDiv.style.top=(topBounceDiv+randomIntFromInterval(0,difTopBottom-circleRadius))+"px"
+    let isStartButtonEnable = true
+    let bounceInterval 
+    let circlesElements = []
+    const bounceDiv = document.getElementById("bounceDiv")
+    const ballCountInput = document.getElementById("ballCountInput")
 
-    console.log(newDiv.style.top)
-    bounceDiv.append(newDiv)
+    const SIXTY_FPS = 1000/60
 
-});
+    return function(){
+        document.getElementById("startButton").addEventListener("click", function(){
+            if(isStartButtonEnable){
+                this.innerText = "Stop"
+                const ballsNumber = parseInt(ballCountInput.value)
+        
+                const {width:widthBounceDiv,height:heightBounceDiv,left:leftBounceDiv,top:topBounceDiv} = bounceDiv.getBoundingClientRect();
+                
+                for(let i=0;i<ballsNumber;i++){
+                    const circleRadius = randomIntFromInterval(40,100)
+                    const newCircle = new Circle(createCircleDomElement(circleRadius),circleRadius,widthBounceDiv,heightBounceDiv,leftBounceDiv,topBounceDiv)
+                    circlesElements.push(newCircle)
+                    addCircleToDom(newCircle.dom,bounceDiv)
+                }
+                
+        
+                bounceInterval = setInterval(()=>{
+                    console.log("Hello")
+                    const {top:topBounceDiv,left:leftBounceDiv,bottom:bottomBounceDiv,right:rightBounceDiv} = bounceDiv.getBoundingClientRect();
+                    circlesElements.forEach((circle)=>{
+                        circle.moveCircle(leftBounceDiv,rightBounceDiv,topBounceDiv,bottomBounceDiv)
+                        updateCirclePositionInDom(circle.dom,circle.x,circle.y)
+                    })
+                },SIXTY_FPS)
+                isStartButtonEnable = !isStartButtonEnable
+            } else {
+                this.innerText = "Bounce"
+                clearInterval(bounceInterval)
+                const removeInterval = setInterval(function(){
+                    circlesElements.forEach((circle)=>{
+                        circle.scaleDown()
+                        updateCircleRadiusInDom(circle.dom,circle.circleRadius)
+                        updateCirclePositionInDom(circle.dom,circle.x,circle.y)
+                    })
+                    circlesElements = circlesElements.filter((circle)=>{
+                        if(circle.circleRadius > 10)
+                        {
+                            return true;
+                        } else {
+                            removeCircleFromDom(circle.dom)
+                            return false;
+                        }
+                    })
+        
+                    if(!circlesElements.length){
```

Comment:
Better be specific and say `circlesElements.length === 0` since `!something` means that something can be a lot of things.

Reply:
Got it